mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/UpiPinFragment.java

```diff
@@ -74,7 +74,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         if (b != null) {
             step = b.getInt(Constants.STEP, 0);
             upiPin = b.getString(Constants.UPI_PIN, null);
-            mTvTitle.setText("Re-enter UPI PIN");
+            mTvTitle.setText(R.string.re_enter_pin);
```

Comment:
Are you sure it is working as expected and displaying the correct text? As I see you are setting the id of the string as the text, I think inside the java code we have to do something like this `getString(R.string.re_enter_pin);`